assets/styles/importer.scss

```diff
@@ -15,6 +15,7 @@
  */
 
 @import 'styles.scss';
+@import 'chris.scss';
```

Comment:
Imported partial `chris.scss` should be written as `chris`<br>Prefer double-quoted strings